packages/video_player/video_player/lib/video_player.dart

```diff
@@ -737,6 +756,7 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     value = value.copyWith(
       position: position,
       caption: _getCaptionAt(position),
+      isCompleted: false,
```

Comment:
I'm not sure whether this is needed/wanted. I do worry that it may cause inconsistencies since the logic that sets `isCompleted` to true also seeks to the end.

Reply:
Can't we pass nil here if `position == duration` to avoid that logic clearing the flag?

Reply:
I set this to just be set to position == duration.